content/en/security/code_security/iast/_index.md

```diff
@@ -104,6 +104,19 @@ Recommendations enable you to change the status of a vulnerability, assign it to
 
 **Note:** To create Jira issues for vulnerabilities, you must configure the Jira integration, and have the `manage_integrations` permission. For detailed instructions, see the [Jira integration][3] documentation, as well as the [Role Based Access Control][4] documentation.
 
+## Vulnerability lifecycle
+
+Datadog automatically manages the lifecycle of vulnerabilities detected by IAST to ensure findings remain accurate and relevant over time.
+
+- **Automatic closure:**
+Vulnerabilities detected by IAST are automatically closed by Datadog when they haven’t been observed for **14 days** since their last detection.
```

Comment:
```suggestion
Vulnerabilities detected by IAST are automatically closed by Datadog when they haven't been observed for **14 days** since their last detection.
```